bundles/de.uka.ipd.sdq.simucomframework/src/de/uka/ipd/sdq/simucomframework/resources/DemandModificationDTO.java

```diff
@@ -0,0 +1,32 @@
+package de.uka.ipd.sdq.simucomframework.resources;
+
+/**
+ * @author Jonas Lehmann
+ *
+ */
+public class DemandModificationDTO {
+
+	private final double scaledDemand;
+
+	private final double additiveDemandValue;
+
+	public DemandModificationDTO(double scaledDemand, double additiveDemandValue) {

```

Comment:
This class does not represent a DTO. We should rename it. --> DemandModification